src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Admin/Index.cshtml

```diff
@@ -16,7 +16,6 @@
             <h4>DateTimeUtc - @ViewBag.DateTimeUtc</h4>
             <div>
                 <a href="/Admin/AddTest" class="btn btn-default">@AddTest</a>
-                <a href="/Admin/CalculateResults" class="btn btn-default">@CalculateResults</a>
```

Comment:
@WTobor Why after deleting this button it is still visible in application?

Reply:
maybe cache? shift+F5 or clean all cookies etc